client/reader/following-manage/subscriptions.jsx

```diff
@@ -124,7 +120,7 @@ class FollowingManageSubscriptions extends Component {
 							sites={ sortedFollows }
 							width={ width }
 							remoteTotalCount={ sortedFollows.length }
-							forceRefresh={ [ feeds, sortedFollows ] }
+							forceRefresh={ sortedFollows }
```

Comment:
i was having trouble convincing react-virtualized to rerender in certain conditions that I can't remember. I'll test out dropping it entirely now